### PR TITLE
Make reverse_unique Django 2.0 compatible

### PR DESCRIPTION
**Pull request by holvianssi**
Fix a couple of places for Django 2.0+ compatibility while keeping tests passing

### CODE REVIEW

File: reverse_unique/fields.py [hunk 1]
```diff
@@ -133,7 +133,7 @@ def get_extra_restriction(self, where_class, alias, related_alias):
         qs = remote_model.objects.filter(self.get_filters()).query
         my_table = self.model._meta.db_table
         rel_table = remote_model._meta.db_table
-        illegal_tables = set([t for t in qs.tables if qs.alias_refcount[t] > 0]).difference(
+        illegal_tables = set([t for t in qs.alias_map.keys() if qs.alias_refcount[t] > 0]).difference(
```
Comment (charettes) [inline reverse_unique/fields.py:136]: Minor but I think the following would work

```suggestion
        illegal_tables = set([t for t in qs.alias_map if qs.alias_refcount[t] > 0]).difference(
```

**Review by charettes — APPROVED** [review]
Glad to see you around Anssi!

I would very much appreciate getting in touch in the next weeks if you'd be interested to chat ORM stuff if that's something you're interested/available with. Just LMK!

File: reverse_unique/fields.py (hunk 1 above)
Comment (holvianssi) [reply reverse_unique/fields.py:136]: Hey, glad to see you! I was planning to do an update for some company internal purposes, but I can work a bit more on this to make this a bit better.

Would you be available for making a new release of this?

**Comment by holvianssi** [comment]
For having a call - yes, definitely, let's do it! I'm having a horrible calendar situation during work hours, and timezones... So how about for example 9pm Helsinki time any day next week or week after that?

**Comment by charettes** [comment]
Going to merge and move to Github actions.

**Comment by charettes** [comment]
Meet call would work fine next week, not sure which day would work best for you though but maybe we could continue the conversation over email? Not sure how to reach out to you, could you contact me over the email associated with my Github account?

**Comment by charettes** [comment]
@holvianssi, #19 made the `master` branch Django 3.2+ compatible but I don't have Pypi upload rights for the `django-reverse-unique` package hence why [I uploaded my own version a while ago](https://pypi.org/project/django-reverse-unique-charettes/).

**Comment by holvianssi** [comment]
Yeah, I believe I have lost my rights to upload packages some time ago. Maybe you could upload a new version to django-reverse-unique-charettes?

**Comment by charettes** [comment]
sounds good, just did upload 1.1 to [django-reverse-unique-charettes](https://pypi.org/project/django-reverse-unique-charettes/)

**Pull request merged**